brjs-sdk/sdk/libs/javascript/br-i18n/src/br/i18n/Translator.js

```diff
@@ -386,6 +386,10 @@ Translator.prototype._getTranslationForKeyOrUndefined = function(token) {
 	token = token.toLowerCase();
 
 	if (!this.tokenExists(token)) {
+		if (typeof window.console === "undefined") {
```

Comment:
Ahh yes ok, misread your comment. My first code snippet (before the 2 edits) is correct then. Something like:

```
var logConsole = (window.jstestdriver) ? jstestdriver.console : window.console;
if (logConsole && logConsole.warn) {
    logConsole.warn('Unable to find a replacement for the i18n key "' + token + '"');
}
```